.travis.yml

```diff
@@ -44,9 +44,6 @@ matrix:
     - os: linux
       language: python
       env: ZSH_BUILD_VERSION=zsh-4.3.11
-    - os: linux
-      language: python
-      env: ZSH_BUILD_VERSION=zsh-4.3.5
```

Comment:
We're going to drop support for 4.3.5 and 4.3.7 on `v2`.

Well, honesty there is little features used that aren't available on 4.3.5/7 (`:A:h` flags and some hooks IIRC) but zsh 4.3.7 is from 2008 so there is little need to keep support for it.